src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/Win32.cs

```diff
@@ -172,7 +171,7 @@ internal static byte[] ConvertIntPtrSidToByteArraySid(IntPtr binaryForm)
             uint length = (uint)SecurityIdentifier.MaxBinaryLength;
             resultSid = new byte[length];
 
-            if (FALSE != Interop.Advapi32.CreateWellKnownSid((int)sidType, domainSid == null ? null : domainSid.BinaryForm, resultSid, ref length))
+            if (FALSE != Interop.Advapi32.CreateWellKnownSid((int)sidType, domainSid?.BinaryForm, resultSid, ref length))
```

Comment:
FWIW: Of the changes in this file, I like this one (reduced the line length to not scroll) and dislike the other two (added horizontal scrolling).